index.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         'has-error-reference': 'error',
         'has-loop-characteristics': 'error',
         'has-message-reference': 'error',
+        'has-no-template': 'error',
```

Comment:
The last concern I have is this name. Can we find a better one that clearly states templates are not supported? As it looks like we settled on `has*` already, so maybe this name is just a thing that users of this plug-in will understand.

Reply:
Any suggestions for a better naming scheme? We can still change it. Right now no one actually gets to see those names.

Reply:
Maybe `templates-not-supported`?

Reply:
Or `no-template-support`?

Reply:
Or `no-templates`.

Reply:
And the opposite would then be `error-reference` instead of `has-error-reference`?

Reply:
I think so. I'd use eslint / [bpmnlint](https://github.com/bpmn-io/bpmnlint/tree/master/rules) as an inspiration that setup this pattern.

Reply:
Shortened it would likely be `no-template` to match bpmnlint exactly:

![image](https://user-images.githubusercontent.com/58601/172809181-5aeac638-11ca-427b-b4a4-12ecb6940548.png)

Reply:
Okay, will adjust.